patches/zeego+1.0.0-alpha.20.patch

```diff
@@ -46,42 +46,29 @@ index f0ae44c..795f9b9 100644
 -  }, title, titleChildren.withoutTargetChildren.filter(child => typeof child != 'string'));
 +    style: style,
 +    className: className
-+  }, titleChildren.withoutTargetChildren.filter(child => typeof child != 'string'), title);
++  }, titleChildren.withoutTargetChildren.filter(child => typeof child != 'string'),title);
  };
  
- const ItemTitle = _ref2 => {
-diff --git a/node_modules/zeego/lib/typescript/menu/web-primitives/index.d.ts b/node_modules/zeego/lib/typescript/menu/web-primitives/index.d.ts
-index 4a69e2e..b5d8045 100644
---- a/node_modules/zeego/lib/typescript/menu/web-primitives/index.d.ts
-+++ b/node_modules/zeego/lib/typescript/menu/web-primitives/index.d.ts
-@@ -1,6 +1,6 @@
- /// <reference types="react" />
- import type { MenuItemProps, MenuItemSubtitleProps, MenuItemTitleProps } from '../types';
--declare const ItemPrimitive: ({ children, style }: MenuItemProps) => JSX.Element;
-+declare const ItemPrimitive: ({ children, style, className }: MenuItemProps & { className?: string }) => JSX.Element;
- declare const ItemTitle: {
-     ({ children, style }: MenuItemTitleProps): JSX.Element;
-     displayName: "ItemTitle";
+ const ItemTitle = menuify(_ref2 => {
 diff --git a/node_modules/zeego/src/menu/web-primitives/index.tsx b/node_modules/zeego/src/menu/web-primitives/index.tsx
-index 86764f4..581e2c7 100644
+index de78759..6dd4331 100644
 --- a/node_modules/zeego/src/menu/web-primitives/index.tsx
 +++ b/node_modules/zeego/src/menu/web-primitives/index.tsx
 @@ -8,7 +8,7 @@ import { pickChildren } from '../children'
  import React from 'react'
- import { MenuDisplayName } from '../display-names'
+ import { menuify } from '../display-names'
  
 -const ItemPrimitive = ({ children, style }: MenuItemProps) => {
-+const ItemPrimitive = ({ children, style, className }: MenuItemProps & { className?:string }) => {
++const ItemPrimitive = ({ children, style, className }: MenuItemProps  & { className?: string }) => {
    const titleChildren = pickChildren(children, ItemTitle)
  
    let title = <></>
-@@ -21,11 +21,12 @@ const ItemPrimitive = ({ children, style }: MenuItemProps) => {
+@@ -21,11 +21,11 @@ const ItemPrimitive = ({ children, style }: MenuItemProps) => {
    }
  
    return (
 -    <View style={style}>
 -      {title}
-+    // @ts-ignore
 +    <View style={style} className={className}>
```

Comment:
is this patch required? I am not sure if passing `className` here would work

Reply:
this is the previous file, I just update the code line and version, and did not do any changes, so it works. 

<img width="501" alt="image" src="https://user-images.githubusercontent.com/37520667/198011954-09134b47-e3a2-46ae-a648-193bd433a1e3.png">